deploy/docker/api.py

```
@@ -453,11 +469,52 @@ async def handle_crawl_request(
+                    results.append(result)
+            else:
+                # Normalize non-async-generator results
+                if results_gen is None:
+                    results = []
+                elif isinstance(results_gen, (list, tuple)):
+                    # It's already a sequence
+                    results = list(results_gen)
+                else:
+                    # Single model/object, wrap it in a list
+                    results = [results_gen]
+        except Exception as bulk_error:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)